### ISSUE TITLE
Ras Machine has unresolved overlaps

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/indra#191]
Maybe due to grounding inconsistencies?
```
Phosphorylation(FYN(), ITPR1(), Y, 353)
Phosphorylation(FYN(), ITPR1())
```

**Comment by bgyori** [comment]
```
Phosphorylation(AMPK(), BRAF()) 
Phosphorylation(AMPK(), BRAF(), S, 429)
```

**Comment by bgyori** [comment]
```
Complex(SOS1(), GRB2()) 
Complex(GRB2(), EGFR(), SOS1()) 
Complex(SHC1(), SOS1(), GRB2())
```

**Comment by bgyori** [comment]
```
Phosphorylation(ERK(), SOS1())
Phosphorylation(MAPK1(), SOS1(), S, 1197)
```

**Comment by bgyori** [comment]
Fixed in 63bb4386131f605030ab0da5cd64fb1782684e2e
